src/js/hooks/productSupplier/form/useProductSupplierValidation.js

```diff
@@ -15,6 +15,14 @@ const useProductSupplierValidation = () => {
   } = useProductSupplierAttributes();
 
   const validationSchema = (data) => {
+    const checkNecessityOfContractPrice = () => {
```

Comment:
What does it mean? The "check necessity"? Check if it is required or not? Sounds weird